File: 0035-search-insert-position/0035-search-insert-position.c
int searchInsert(int* nums, int numsSize, int target) {
    if(nums[0]>=target)
    {
        return 0;
    }
   
    for(int i=0;i<numsSize;i++)
    {
        if(nums[i]==target)
        {
            return i;
        }
    }
    for(int i=0;i<numsSize-1;i++)
    {
        if(nums[i]<target && nums[i+1]>target)
        {
            return i+1;
        }
    }

    return numsSize;
}

File: 0035-search-insert-position/0035-search-insert-position.java
class Solution {
    public int searchInsert(int[] nums, int target) {
        if(nums[0]>=target)return 0;
        else{
             for(int i=0;i<nums.length;i++)
             {
                if(nums[i]==target)return i;
             }
             for(int i=0;i<nums.length-1;i++)
             {
                if(nums[i]<target && nums[i+1]>target)return i+1;
            }
        }
        return nums.length;
    }
}